src/Components/CreateImageWizard/CreateImageWizard.js

```diff
@@ -121,6 +121,7 @@ const onSave = (values) => {
         },
       ],
       customizations,
+      client_id: 'ui',
```

Comment:
Why is the value always ui?

Reply:
The result is to be able to differentiate between requests created via api and requests created via ui, Since all of these requests are coming from ui, the client id for all of them is the same.

Reply:
thanks!